controllers/rcj_soccer_referee_supervisor/rcj_soccer_referee_supervisor.py

```diff
@@ -12,7 +12,9 @@
 automatic_mode = True if "RCJ_SIM_AUTO_MODE" in environ.keys() else False
 
 TEAM_YELLOW = environ.get("TEAM_YELLOW_NAME", "The Yellows")
+TEAM_YELLOW_ID = environ.get("TEAM_YELLOW_ID")
 TEAM_BLUE = environ.get("TEAM_BLUE_NAME", "The Blues")
+TEAM_BLUE_ID = environ.get("TEAM_BLUE_ID")
```

Comment:
We should assign these IDs some default values, otherwise the `output_prefix` path would contain `None`.